packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_counter/di/counter_dependencies.dart

```diff
@@ -17,15 +27,28 @@ class CounterDependencies {
 
   final BuildContext context;
 
-  /// List of all providers used throughout the app
   List<SingleChildWidget> get providers => [
+        ..._dataSources,
         ..._repositories,
         ..._blocs,
       ];
 
+  /// In case your application use API and HttpClient(dio and retrofit
+  /// by default), here should uncomment the row with CountRemoteDataSource
+  /// and comment current "create" function, witch use CountLocalDataSource.
+  /// Also you should provide a real api in
+  /// lib\base\count_data_source\count_remote_data_source.dart
```

Comment:
@RomanovaPrime, the same comment regarding the filenames as above.

Reply:
Edited.